notebooks/en/enterprise_cookbook_gradio.ipynb

```diff
@@ -0,0 +1,5796 @@
+{
```

Comment:
<p>Might be nice to include a link to Gradio here as it is the first mention of it in case users want to learn more</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>